pkg/client/client.go

```diff
@@ -38,17 +38,17 @@ import (
 )
 
 type AuthOpts struct {
-	AuthURL          string                   `gcfg:"auth-url" mapstructure:"auth-url" name:"os-authURL" dependsOn:"os-password|os-trustID|os-applicationCredentialSecret|os-clientCertPath"`
+	AuthURL          string                   `gcfg:"auth-url" mapstructure:"auth-url" name:"os-authURL" dependsOn:"os-password|os-trustID|os-applicationCredentialSecret|os-applicationCredentialName|os-clientCertPath"`
```

Comment:
Is there a need to include `applicationCredentialID` here as well?

Reply:
Yes, you are correct. I updated the PR.